templates/site/ide.html

```diff
@@ -30,6 +30,23 @@
             function nodeFilter([_name, data]) {
                 return data.services && data.services.some(service => service.keywords.includes("jupyterhub"))
             }
+
+            function setNodeLinkJupyter(name, data, node_id_suffix) {
+                let nodeLinkID = "link-url-" + node_id_suffix;
+
+                data.services.forEach((service) => {
+
+                    if(service.keywords.includes("jupyterhub")) {
+                        service.links.forEach((link) => {
+
+                            if(link.href.includes("jupyter") && link.rel == "service" ) {
```

Comment:
`link.href.includes("jupyter")` is unnecessary and may lead to errors if a node has a non-standard URL for their jupyterhub landing page

Reply:
use `===` comparison whenever possible